ovirt-ansible-collection.spec.in

```diff
@@ -13,6 +13,7 @@ BuildArch:      noarch
 Url:            http://www.ovirt.org
 
 Requires: ansible >= 2.9.0
+Requires: python3-ovirt-engine-sdk4
```

Comment:
Shouldn't we also here require 4.4 SDK?
```Requires: python3-ovirt-engine-sdk4 >= 4.4.0```

Reply:
I was not sure with it but would be best to implement it.